common/filestore_utils.py

```diff
@@ -26,7 +29,23 @@ def _using_gsutil():
     except KeyError:
         return True
 
-    return experiment_filestore_path.startswith('gs://')
+    return is_filestore_path_gcs(experiment_filestore_path)
+
+
+def is_filestore_path_gcs(filestore_path):
```

Comment:
usually we name like is_gcs_filestore_path ?

Reply:
Done.